doc/developer-guides/security-hld.rst

```diff
@@ -0,0 +1,1079 @@
+.. _security-hld:
+
+Security high-level design
+##########################
+
+.. primary author: Bing Zhu
+   contributor: Yadong Qi
+
+Introduction
+************
+
+This document describes the ACRN security high level design on Apollo
+Lake SoCs, including information about:.
```

Comment:
There is a dot right after the semi-colon, is that intended?

Reply:
@dbkinder @bzhu5 to address the comment

Reply:
@dbkinder you can remove that "dot", thank you